lib/MusicBrainz/Server/Entity/LinkType.pm

```diff
@@ -149,6 +151,7 @@ around TO_JSON => sub {
     $json->{reverse_link_phrase} = $self->reverse_link_phrase;
     $json->{type0} = $self->entity0_type;
     $json->{type1} = $self->entity1_type;
+    $json->{children} = \@children if @children;
```

Comment:
I think this should be fine, though it duplicates work done by https://github.com/reosarevok/musicbrainz-server/blob/MBS-10740/lib/MusicBrainz/Server/Form/Utils.pm#L179 so we should now remove the line I linked.